database/migrations/2024_08_29_061841_add_importer_job_system_modules.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('system_modules', function (Blueprint $table) {
+            //
+            $table->string('importer_job')->nullable()->after('bulk_actions');
```

Comment:
Because each system module have his own importer, Example company and products are different but i don't want to create a table for storage of it, o hard code the code.